packages/current-albatross-deployer/current-albatross-deployer.1.0.0/opam

```diff
@@ -0,0 +1,66 @@
+opam-version: "2.0"
+maintainer:   "Lucas Pluvinage"
+authors:      "Lucas Pluvinage"
+license:      "ISC"
+homepage:     "https://github.com/tarides/current-albatross-deployer"
+bug-reports:  "https://github.com/tarides/current-albatross-deployer/issues"
+dev-repo:     "git+https://github.com/tarides/current-albatross-deployer.git"
+doc:         "https://tarides.github.io/current-albatross-deployer/"
+
+depends: [
+  "albatross" {>= "1.5.1"}
+  "obuilder-spec" {>= "0.5"}
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.9.0"}
+  "odoc" {with-doc}
+  "asn1-combinators" {>= "0.2.6"}
+  "bos" {>= "0.2.0"}
+  "cmdliner" {>= "1.1.0"}
+  "cstruct" {>= "6.0.1"}
+  "current" {>= "0.5"}
+  "current_docker" {>= "0.5"}
+  "current_web" {with-test}
+  "ipaddr" {>= "5.2.0"}
+  "logs" {>= "0.7.0"}
+  "lwt" {>= "5.4.2"}
```

Comment:
This should be
```suggestion
  "lwt" {>= "5.6.0"}
```
to satisfy the lower bounds checking on `Lwt_result.map_error`. See https://ocaml.org/p/lwt/5.6.1/doc/Lwt_result/index.html#val-map_error for the deprecation / introduction